I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00keymap.rpy", line 473, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 473, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 492, in _execute_python_hide
    renpy.save("_reload-1", "reload save game")
AttributeError: 'NoneType' object has no attribute 'stores'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00keymap.rpy", line 473, in script
    python hide:
  File "E:\renpy-7.3.2-sdk to home\renpy-7.3.2-sdk\renpy\ast.py", line 1111, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "E:\renpy-7.3.2-sdk to home\renpy-7.3.2-sdk\renpy\python.py", line 1049, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00keymap.rpy", line 473, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 492, in _execute_python_hide
    renpy.save("_reload-1", "reload save game")
  File "E:\renpy-7.3.2-sdk to home\renpy-7.3.2-sdk\renpy\loadsave.py", line 382, in save
    roots = renpy.game.log.freeze(None)
  File "E:\renpy-7.3.2-sdk to home\renpy-7.3.2-sdk\renpy\rollback.py", line 1017, in freeze
    self.complete(False)
  File "E:\renpy-7.3.2-sdk to home\renpy-7.3.2-sdk\renpy\rollback.py", line 584, in complete
    self.current.stores[name], self.current.delta_ebc[name] = delta
AttributeError: 'NoneType' object has no attribute 'stores'

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.0.22062402
Moment Of Eternity 0.03
Fri Jul  8 17:38:47 2022
